docs/extend/plugins_authorization.md

```diff
@@ -104,6 +104,7 @@ support the Docker client interactions detailed in this section.
 Enable the authorization plugin with a dedicated command line flag in the
 `--authorization-plugin=PLUGIN_ID` format. The flag supplies a `PLUGIN_ID`
 value. This value can be the plugin’s socket or a path to a specification file.
+The authorization plugin can be reloaded without requiring a [restart](../reference/commandline/dockerd.md).
```

Comment:
The link should point to the "configuration reloading" section; can you add `#configuration-reloading` to the link?

I'm looking for a better way to describe the reloading, because now it may feel as if we reload/restart the *plugin* (but perhaps that's me), would this be better? (not sure it is, so open to suggestions):

```markdown
Authorization plugins can be loaded without restarting the daemon. Refer
to the [`dockerd` documentation](../reference/commandline/dockerd.md#configuration-reloading) for more information.
```